src/Dynamo.All.sln

```diff
@@ -1013,4 +1013,4 @@ Global
 	GlobalSection(ExtensibilityGlobals) = postSolution
 		SolutionGuid = {89CB19C6-BF0A-4E6A-BFDA-79D143EAB59D}
 	EndGlobalSection
-EndGlobal
```

Comment:
I see this end of line change as well when building Dynamo. Do you guys see it?

Reply:
I do not see this.

Reply:
ok, could be my VS settings then. Will revert this change.